src/main.test.js

```diff
@@ -1,8 +1,8 @@
-const envSpecToHTML = require("./main.js");
+const envSpec = require("./main.js");
 
 test("Valid input with type", () => {
   const testEnv = "DATABASE_URL\nADMIN_EMAIL:email";
-  expect(envSpecToHTML(testEnv)).toEqual(
+  expect(envSpec.parse(testEnv)).resolves.toEqual(
```

Comment:
You need to `return` this, in order to mark the test successfully as passed/failing ([see example](https://jestjs.io/docs/en/expect.html#resolves)).